docs/admin/multiple-schedulers.md

```diff
@@ -0,0 +1,236 @@
+---
+---
+
+Kubernetes ships with a default scheduler that is described [here](kube-scheduler.md).
+If the default scheduler shipped with Kubernetes does not suit your needs you can
+implement your own scheduler. Not just that, you can even run multiple schedulers
+simultaneously alongside the default scheduler and instruct Kubernetes what scheduler to
+use for each of your pods. Let's learn how to run multiple schedulers in Kubernetes with
+an example.
+
+Detailed description of how to implement a scheduler is outside the scope of this
+document. Please refer to the kube-scheduler implementation in
+[plugin/pkg/scheduler](https://github.com/kubernetes/kubernetes/tree/master/plugin/pkg/scheduler)
+in the Kubernetes source directory for a canonical example.
+
+### 1. Package the scheduler
+
+Package your scheduler binary into a container image. For the purposes of this example,
+let's just use kube-scheduler as our second scheduler as well. Clone the
+[Kubernetes source code from Github](https://github.com/kubernetes/kubernetes)
+and build the source.
+
+```shell
+$ git clone https://github.com/kubernetes/kubernetes.git
+$ cd kubernetes
+$ hack/build-go.sh
+```
+
+Copy the built kube-scheduler binary into the container image. Here is the `Dockerfile`
```

Comment:
For first sentence I would say "Create a container image containing the kube-scheduler binary."